### DISCUSSION TITLE
Welcome to mobx Discussions!

### DISCUSSION THREAD

**Discussion by mweststrate** [mobxjs/mobx#2406] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by kubk** [comment, marked as answer]
Is it going to replace gitter and spectrum mobx chats?

**Reply by mweststrate** [reply]
Yeah if it works well :) I think it is easier to keep an eye on and relate to issues :). But giving it first a small try as it is brand new
